modules/core/core-api/src/main/java/com/enonic/xp/image/ImageHelper.java

```diff
@@ -203,6 +241,18 @@ public static BufferedImage scaleSquare( final BufferedImage source, final int s
         return getScaledInstance( cropped, size, size );
     }
 
+    public static BufferedImage scaleWidth( BufferedImage source, final int size )
```

Comment:
newWidth instead of size
API should have more generic method. This one is too specific